plugins/workflow/category/category.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extension type="plugin" group="workflow" method="upgrade">
+	<name>plg_workflow_category</name>
+	<author>Joomla! Project</author>
+	<creationDate>2025-06</creationDate>
+	<copyright>(C) 2025 Open Source Matters, Inc.</copyright>
+	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+	<authorEmail>admin@joomla.org</authorEmail>
+	<authorUrl>www.joomla.org</authorUrl>
+	<version>6.0.0</version>
+	<description>PLG_WORKFLOW_CATEGORY_DESC</description>
+	<namespace path="src">Joomla\Plugin\Workflow\Category</namespace>
+	<files>
+		<folder>forms</folder>
+		<folder>language</folder>
+		<folder>src</folder>
+		<folder plugin="category">services</folder>
+	</files>
+	<languages>
+		<language tag="en-GB">language/en-GB/plg_workflow_category.ini</language>
+		<language tag="en-GB">language/en-GB/plg_workflow_category.sys.ini</language>
+	</languages>
+	<config>
+		<fields name="params">
+			<fieldset name="basic">
+				<field
+					name="default_category"
+					type="category"
+					extension="com_content"
+					label="PLG_WORKFLOW_CATEGORY_LABEL"
+				/>
```

Comment:
Thinking about this, I came to the conclusion, it makes no sense to have a default category. I mean, when in the transition no category is selected, no chance should happen at all (not all transitions neeed a category change)